packages/core/src/components/Pagination/Select.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import clsx from "clsx";
+// import { withStyles } from "@mui/styles";
+import {
+  HvBaseDropdown,
+  HvSelectionList,
+  HvListItem,
+  HvPanel,
+  HvTypography,
+} from "../index";
+import { StyledRoot } from "./Select.styles";
+
+export const Option = ({ children, ...others }) => (
+  <HvListItem {...others}>{children}</HvListItem>
+);
+
+const HvSelect = ({
```

Comment:
In [DS5](https://designsystem.hitachivantara.com/6c705d900/p/31a72a-pagination/b/82dd7c/i/190304763), the select seems to have a border with some radius.